### PR TITLE
Checkbox issue fix 

### LINKED ISSUE #57: Checkbox not updating in formValue
When i check a checkbox its not adding that to formValue i tried printing it and it doesnt adds that 

so it should have added selling 1 when i checked the checkbox

These are key and value of formValue

newname xyz
apply_on Item Code
items [{item_code: 1101, uom: Kg}]
item_groups null
brands null
apply_rule_on_other null
other_item_code null
other_item_group null
other_brand null
applicable_for Customer
customer M Y
customer_group null
territory null
sales_partner null
campaign null
supplier null
supplier_group null
valid_from 2021-07-24T00:00:00.000
valid_upto 2021-07-27T00:00:00.000
company Abc
currency INR
price_discount_slabs [{min_qty: 5, max_qty: 5, min_amount: 0, max_amount: 0, rate_or_discount: Rate, rate: 5}]

### PR DESCRIPTION
**Pull request by mohit7208**
Check box issue fix #57 the only problem i am facing was that validator type the formbuilder package had of type String Function(bool) and in code it was List<String? Function(dynamic?)> so i commented it can you guide me on how to do that
Checkbox value are reflecting in the form when saving a form

### CODE REVIEW

File: lib/widgets/custom_form_builder_checkbox.dart [hunk 1]
```diff
@@ -5,155 +5,301 @@ import 'package:flutter/widgets.dart';
...
-  final bool initialValue;
-  final bool readOnly;
-  final InputDecoration decoration;
-  final ValueChanged onChanged;
-  final ValueTransformer valueTransformer;
-  final bool leadingInput;
+// class CustomFormBuilderCheckbox extends StatefulWidget {
+//   final String attribute;
+//   final List<FormFieldValidator> validators;
+//   final bool initialValue;
```
Comment (sumitbhanushali) [inline lib/widgets/custom_form_builder_checkbox.dart]: please remove commented code

**Comment by sumitbhanushali** [comment]
@mohit7208 check how validator is added in other controls. check `data.dart` file for clarity

File: lib/widgets/custom_form_builder_checkbox.dart (hunk 1 above)
Comment (mohit7208) [reply lib/widgets/custom_form_builder_checkbox.dart]: > please remove commented code

removed

**Comment by mohit7208** [comment]
> @mohit7208 check how validator is added in other controls. check `data.dart` file for clarity

@sumitbhanushali  done that

**Pull request merged**